tests/cache-manager-test.js

```diff
@@ -0,0 +1,136 @@
+'use strict';
+
+const assert = require('assert');
+const sandbox = require('sinon').createSandbox();
+const mockRequire = require('mock-require');
+
+/* const memoryMock = require('../lib/mocks/lru-cache-mock');
+
+mockRequire('LRU', memoryMock);
+const MemoryManager = require('../lib/memory-manager'); */
+const CacheManager = require('../index');
+const { CacheManagerError } = require('../lib');
+
+describe('Cache Manager Test', () => {
+
+	let cache;
+
+	before(() => {
+		cache = new CacheManager('tests');
+	});
+
+	after(() => {
+		cache.reset();
+		cache.redis.close();
+	});
+
+	context('manipulating data', () => {
```

Comment:
```suggestion
	context('when manipulating data', () => {
```
En `context()` se utiliza el `when` para describir un contexto que involucre a mas de un `it()` esto hace que sea mas comprensible la lectura de los tests. Esto aplica en varias ocasiones de los tests